### PR TITLE
Added 'schema_changes_from_baseline' to the test recommendations model.

### PR DESCRIPTION
**Pull request by elongl**

### CODE REVIEW

File: elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql [hunk 1]
```diff
@@ -72,14 +82,23 @@ with
...
+            end as table_args
         from pending_recommended_tests
         join tables_criticality using (id)
         left join timestamp_columns using (database_name, schema_name, table_name)
-        where requires_timestamp_column = false or timestamp_column is not null
+        left join table_columns using (database_name, schema_name, table_name)
+        where
+        short_name = 'volume_anomalies'
+        or
+        short_name = 'freshness_anomalies' and timestamp_column is not null
```
Comment (ofek1weiss) [inline elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql:99]: add  `()` around

File: elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql [hunk 2]
```diff
@@ -72,14 +82,23 @@ with
...
+        or
+        short_name = 'schema_changes_from_baseline' and table_columns.columns is not null
```
Comment (ofek1weiss) [inline elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql:101]: here too

File: elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql (hunk 1 above)
Comment (elongl) [reply elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql:99]: Why? `and` evaluates before `or`.

Comment (ofek1weiss) [reply elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql:99]: im not sure about it, i believe its just in sequential order, but thats exactly the point - it just more explicit with `()` (i like explicit)

**Review by ofek1weiss — APPROVED** [review]

**Pull request merged**